app/assets/stylesheets/refills/_hero.scss

```diff
@@ -0,0 +1,64 @@
+.hero {  
+  ///////////////////////////////////////////////////////////////////////////////////
+  $base-border-radius: 3px !default;
+  $base-accent-color: #477DCA !default;
+  $large-screen: em(860) !default;
+  //////////////////////////////////////////////////////////////////////////////////
+
+  $hero-background-top: #7F99BE; 
```

Comment:
Color `#7F99BE` should be written as `#7f99be`<br>Line contains trailing whitespace